pytorch3d/renderer/mesh/rasterizer.py

```diff
@@ -198,10 +198,6 @@ def transform(self, meshes_world, **kwargs) -> torch.Tensor:
         verts_view = cameras.get_world_to_view_transform(**kwargs).transform_points(
             verts_world, eps=eps
         )
-        # Call transform_points instead of explicitly composing transforms to handle
-        # the case, where camera class does not have a projection matrix form.
-        verts_proj = cameras.transform_points(verts_world, eps=eps)
-        to_ndc_transform = cameras.get_ndc_camera_transform(**kwargs)
```

Comment:
Did you mean to leave this line in and only remove the previous line? If so, then I think the fix is right.

Reply:
Yes, sorry for that. I only meant to remove the line containing the camera.transform_points() and the comment, as it makes more sense in the else part.

Reply:
I created a second commit in the branch to resolve the issue. Let me know if you want a new pull request.

Reply:
Created a new pull request where the issue is fixed.